packages/hurumap-next/src/LocationTag/LocationTag.js

```diff
@@ -0,0 +1,142 @@
+import { Box, LinearProgress, Typography, IconButton } from "@mui/material";
+import { alpha } from "@mui/material/styles";
+import { capitalize } from "@mui/material/utils";
+import makeStyles from "@mui/styles/makeStyles";
+import clsx from "clsx";
+import React from "react";
+
+import { ReactComponent as CancelIcon } from "../assets/icons/cancel.svg";
+
+const useStyles = makeStyles((theme) => {
+  const { palette, typography } = theme;
```

Comment:
Part of this migration is moving away from `useStyles` which is [deprecated](https://mui.com/system/styles/basics/) (doesn't support React 18, etc.).